source/jsonnet/northern-ireland/lib/common_rules.libsonnet

```diff
@@ -63,7 +63,7 @@
     id: 'date-of-birth-answer',
     condition: 'greater than',
     date_comparison: {
-      value: '2021-07-01',
+      value: '2020-07-01',
```

Comment:
Does this not mean. the over5 also needs changing.
Current:
```js
schoolYear5OrOver: {
    id: 'date-of-birth-answer',
    condition: 'less than or equal to',
    date_comparison: {
      value: '2021-07-01',
      offset_by: {
        years: -5,
      },
    },
  },
```
5 or Over looks like `50` -_-

Reply:
<img width="716" alt="Screenshot 2020-07-08 at 11 43 27" src="https://user-images.githubusercontent.com/53475968/86909708-68cb1a80-c110-11ea-9194-382c70c17255.png">

Reply:
I mean suggestions for something better would help, maybe `schoolYear5AndOver`?

Reply:
That works 👍

Reply:
schoolYear5AndOver works for me, as it highlights the double condition. I read it as 50 and over and didn't go near it